llcmapping.py

```diff
@@ -26,8 +25,8 @@ def __init__(self, ds, dx=0.25, dy=0.25):
         self.new_grid  = pyresample.geometry.GridDefinition(lons=self.new_grid_lon,
                                                             lats=self.new_grid_lat)
 
-    def __call__(self, da, ax=None, projection='robin', lon_0=-60, **plt_kwargs):
-        
+    def __call__(self, da, ax=None, projection=cart.crs.PlateCarree(), lon_0=-60, **plt_kwargs):
+
```

Comment:
Can we use `cart.crs.Robinson()` here instead?

Reply:
sure! changed